### ISSUE TITLE
text‐scaling‐factor cannot be set from command line

### ISSUE THREAD

**Issue by Sadi58** [linuxmint/Cinnamon#7262]
```
 * Cinnamon 3.6.7
 * Mint 18.3
 * Intel Skylake
 * 64 bit
 ```

**The issue**
The command `gsettings set org.cinnamon.desktop.interface text‐scaling‐factor x.x` does not work. The output is "`No such key 'text‐scaling‐factor'`" although there is such a key and it immediately takes effect if the value is changed via **dconf-editor** or **System Settings** > **Fonts** > **Text scaling factor**.

**Steps to reproduce**
Enter `gsettings set org.cinnamon.desktop.interface text‐scaling‐factor 1.2` in terminal.

**Expected behaviour**
It should immediately change the setting accessible via **dconf-editor** or **System Settings** > **Fonts** > **Text scaling factor**.

**Other information**
This is important in handling certain issues related with HiDPI configuration in a scenario where user has a laptop which is sometimes connected to a external desktop monitor where a startup script is required to make some adjustments for the laptop display or the external monitor.

**Comment by JosephMcc** [comment]
So weirdly if I copy/paste your text I get that error. However if I type it myself it works just fine.

**Comment by leigh123linux** [comment]
@JosephMcc  It fails here on fedora if I copy and paste it, the only difference is character encoding.
I doubt the issue is in the cinnamon code.

**Comment by JosephMcc** [comment]
I don't really see this being a cinnamon issue either. Especially since it works here on git master. I'll reopen it if there is some more info providing showing the issue is actually with Cinnamon.

**Comment by Odyseus** [comment]
Hello, everybody.

@Sadi58: You are using the wrong character on text"‐"scaling"‐"factor. You are using the **Hyphen** character (UTF-8 representation 0xE2 0x80 0x90), when you should be using the **Hyphen-minus** character (UTF-8 representation 0x2D).

In short, `text‐scaling‐factor` is not equal to `text-scaling-factor`.

**Comment by Sadi58** [comment]
Thank you all!!!
And sorry for taking your time :-(
What a wierd thing. I copied this command from a website which as in English and pasted it into my login startup script.
When it didn't worked I copied and pasted it to terminal and saw that puzzling message  "No such key 'text‐scaling‐factor'".
I also tested this under Ubuntu 16.04 live session without any problem (but I had to enter the text manually that time ;-)
So I jumped to the conclusion that it might be a Cinnamon bug.
I should have tested it under a Linux Mint Live Session as well ( embarrassed emo here ; -)
Oh, these strange character and character encoding problems, they catch me unprepared from time to time...
